test/evergreen.yml

```diff
@@ -1614,6 +1621,25 @@ tasks:
               cp -rf WT_TEST WT_TEST_$file
             done
 
+  - name: time-shift-sensitivity-test
+    depends_on:
+      - name: compile-enable-strict
+    commands:
+      - func: "fetch artifacts"
+        vars:
+          dependent_task: compile-enable-strict
+      - command: shell.exec
+        params:
+          working_dir: "wiredtiger/build_posix/test/csuite"
+          script: |
+            set -o errexit
+            set -o verbose
+
+            rundir=../../../test/csuite
+            cp $rundir/time_shift_test.sh ./
+            chmod a+x ./time_shift_test.sh
+            ./time_shift_test.sh /usr/local/lib/faketime/libfaketimeMT.so.1 0-1
```

Comment:
The script copying code here makes me sick :) 
I know it's from the Jenkins job, and suggest to try and see if we can call the script directly without copying it over, or if the copying is necessary see if it's possible to improve the script to make it runnable under any directory.

Reply:
I tried to run the script(time_shift_test.sh) directly without copying but it internally calls another script (test_rwlock) which is located at the copied to location. I will discuss with you if I can change the script.

Reply:
Modified the script to run without copying to some location.